python/pyspark/ml/torch/ssh_manager.py

```diff
@@ -0,0 +1,55 @@
+import os
+import subprocess
+import shutil
+from typing import (List)
+
+from pyspark.ml.torch.utils import write_to_location
+
+class SSHEnvManager:
```

Comment:
Which methods of this class will be called from the DeepspeedDistributor? Can we make the other ones private (add leading _)